From 5c6968b9d8842010f68eccbc57de8b54dcab5a6d Mon Sep 17 00:00:00 2001
From: Matthew James Briggs <matthew.james.briggs@gmail.com>
Date: Wed, 5 May 2021 19:57:11 -0700
Subject: [PATCH] use local schema location

The network schema locations are not what we want, instead we want to
locate these schemas on disk.
---
 schema/musicxml.xsd | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/schema/musicxml.xsd b/schema/musicxml.xsd
index 48116bf..7f901a5 100755
--- a/schema/musicxml.xsd
+++ b/schema/musicxml.xsd
@@ -20,8 +20,8 @@ This file defines the MusicXML 3.0 XSD, including the score-partwise and score-t
 	<xs:annotation>
 		<xs:documentation>The MusicXML 3.0 DTD has no namespace, so for compatibility the MusicXML 3.0 XSD has no namespace either. Those who need to import the MusicXML XSD into another schema are advised to create a new version that uses "http://www.musicxml.org/xsd/MusicXML" as the namespace.</xs:documentation>
 	</xs:annotation>
-	<xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="http://www.musicxml.org/xsd/xml.xsd"/>
-	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="http://www.musicxml.org/xsd/xlink.xsd"/>
+	<xs:import namespace="http://www.w3.org/XML/1998/namespace" schemaLocation="./xml.xsd"/>
+	<xs:import namespace="http://www.w3.org/1999/xlink" schemaLocation="./xlink.xsd"/>
 
 	<!-- Simple types derived from common.mod entities and elements -->
 
-- 
2.24.3 (Apple Git-128)

